content/en/docs/concepts/services-networking/topology-aware-hints.md

```diff
@@ -19,12 +19,6 @@ those network endpoints can be routed closer to where it originated.
 For example, you can route traffic within a locality to reduce
 costs, or to improve network performance.
 
-{{< note >}}
-The "topology-aware hints" feature is at Beta stage and it is **NOT** enabled
-by default. To try out this feature, you have to enable the `TopologyAwareHints`
-[feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
-{{< /note >}}
```

Comment:
Not sure what the standard is here, but maybe we should clarify that the feature graduated to beta in 1.23 and the feature gate was enabled by default on 1.24+. Note that even when the feature gate is enabled, users still need to set the annotation to enable the feature on a Service.

Reply:
The shortcode we have doesn't capture a separate “enabled by default from release” value. Something we could add, or track as an enhancement.

Reply:
@robscott I guess information about annotation should be added here https://github.com/kubernetes/website/issues/36830
@sftim I can also make PR to fix this

Reply:
About “enabled by default from release”
I guess that it is very useful information. But it should be tracked separately (I also will be happy to work on it)